data/korath/korath ships.txt

```diff
@@ -15,58 +15,57 @@ ship "Korath Dredger"
 	attributes
 		category "Heavy Freighter"
 		"cost" 29370000
-		"shields" 39000
-		"hull" 16000
-		"required crew" 201
-		"bunks" 395
-		"mass" 1503
-		"drag" 18.3
-		"heat dissipation" .65
+		"shields" 27000
+		"hull" 22000
+		"required crew" 137
+		"bunks" 331
+		"mass" 1357
+		"drag" 17.1
+		"heat dissipation" .45
 		"fuel capacity" 900
 		"cargo space" 605
-		"outfit space" 827
-		"weapon capacity" 297
-		"engine capacity" 181
-		
+		"outfit space" 607
+		"weapon capacity" 226
```

Comment:
```suggestion
		"weapon capacity" 276
```
I'll just point out that this change shifts the Dredger down from 3 less than the Bactrian to 74 less weapon space than the Bactrian. I mention the Bactrian specifically because the Dredger has, so far, been portrayed as being something of a similar role: Basically a heavy freighter that is built to have a substantial permanent population onboard, and is setup to be able to handle regular combat duties. 

In short, the Bactrian is a general-purpose ship that is basically one of if not *the* best solo ship in the game. The Dredger is built for the same purpose, by a faction that has significantly better technology and an active need for this kind of ship. (unlike the Bactrian, the majority of whose role was rendered obsolete centuries ago). As such, the Dredger was originally intended to be basically an upgrade to the Bactrian so that people who wanted that style of ship would have something to look forward to in the mid game. Nerfing its combat capabilities too much (especially given the rather massive size of Korath outfits) is likely to render this ship unattractive to many, perhaps all, those who would like alternatives to the Bactrian.

Reply:
While I agree with most of your reverts or edits, the weapon one was our most specific nerf. We didn't want the Dredger to be able to carry anything larger than a load of 2 GS, 3 Diggers, 1 Warder and 1 Banisher, but here you are giving it room to do so which defeats the purpose of the nerf. Yes it might be more comparable to the bactrian with your changes, but that isn't our intention. 

In fact, addressing your statements of the dredger being a big crew ship, we are moreso stepping back from a generalized classification of that. It's more of a utility vessel with many bunks and more cargo than it is either a warship with a lot of them, or a city ship with a lot of them. The main role is mostly mining now, and to support Raiders now, so in that regard I don't agree with rebuffed the weapon space due to bactrian comparisons. 

Any idea to purely compare it to the bactrian or as a crew vessel we are now delegating to the upcoming KWS changes and variants. Since a good chunk of the nerfs is from warp core I suggest waiting on his input too.

Reply:
But yeah, we are trying to take the Dredger a little bit out of the spotlight as an all-use vessel and more as a utility vessel, as all the new kws variants will definitely take the spotlight for multipurpose.

Reply:
I meant "regular combat duties" as in "it is expected to be a support ship in fleets that regularly engage in combat; not "it is expected to be a frontline combatant." As in, it's expected to see combat as a regular occurance, albeit it should be the one keeping stores safe and providing supporting fire while the actual warships drive off and/or disable the enemy.

This should be the ship that lags behind while the warships blaze ahead to engage, preferably engaging only from a distance. (I'd rather like it if we could get the Korath some 400mm railguns like the Aurorans made good use of in Escape Velocity Nova. That would be the ideal sort of weaponry for the Dredger's preferred combat role.

> Then they should go use an actual HW instead of a Transport ship if they want good combat capabilities.

I keep telling various people that in regards to the Bactrian, but here we are with it still being in the running for best Heavy Warship while also being in the running for best transport, best freighter, most versatile ship, and still manages to do suprisingly well in terms of manoeuvrability. If we can't beat'm, join'm and at least give people an alternative.

> But yeah, we are trying to take the Dredger a little bit out of the spotlight as an all-use vessel and more as a utility vessel, as all the new kws variants will definitely take the spotlight for multipurpose.

As I see it right now, the Dredger as-is occupies something of a sweet spot in that it is big enough to be a solo ship, but not so big that it can't be risked flying without a fleet. Contrast with the KWS and other huge ships that, frankly, I don't see the Korath ever risking going anywhere remotely dangerous without a large fleet of escorts. Too valuable. Ultra-useful, yes, and they'll put the Dredger permanently in the shadows when/if the player ever manages to get one. But that's no reason to erase the Dredger's versatility in the meantime.

Anyway, the point, for me, is that the Korath should basically have a whole range of Bactrians. Their whole civilization lives onboard ships. They should have a Bactrian-style ship in basically every category, and the Dredger is the one for the "Heavy Freighter" segment. I don't know if we'll ever actually *get* all those ships, but even if not visible that's what I'd expect them to have.

Reply:
While I think that simply having limited offense in variants is doable while bumping up stats again, we we have to see what @warp-core has to say as he is most adamant about the dredgers space changes and limitations and did so in respect to the bactrian.

Reply:
I'm looking forward to hearing them. 

Honestly, from what I'm hearing so far, this is sounding like a turning the dredger from a Exiles ship into a Efret ship. It makes sense for the Efret to have ships that aren't particularly well suited to combat, since they are potentially trying to appear non-aggressive and peaceful for the Quarg. It doesn't make sense for the Exiles, who are expressly militant.

But that being said, I'm glad you like the idea of simply ensuring there aren't any variants outfitted that way. A "soft-block", so to speak, against heavy-offense dredgers instead of a hard-block.

Reply:
> Basically a heavy freighter that is built to have a substantial permanent population onboard, and is setup to be able to handle regular combat duties.
This sounds like a role that at least one, probably more, of hte planned World-Ship flavours will fill.

The old stats, and the stats you've suggested (granted, to a lesser extent) don't have it as a support ship for groups of Raiders, prior to the recent Raider buffs, it would have probably been the most powerful ship in those groups. The battleship at the heart of the fleet, that also serves as the cargo/personnel support ship, with Raiders supporting it.
Given that buffing the Raider wasn't done just so it'd be better for combat than the Dredger (and, if that was hte intention, that's just power creep), so I don't think that the fact that the Dredger comes in a little below the Raider now is enough to justify these stats for the *support ship*, where combat is a tertiary aim (after cargo and personnel transport).

I think 757 outfit and 276 weapon space is too much, Maybe around 675/240 if it absolutely needs more, but I think this serves the role of heavy freighter and transport with the ability to provide support to Raiders in battle (I would actually prefer it have gun ports with Firelights for that purpose, though), instead of being a battleship that happens to have some raiders with it.

Reply:
Re Saugia:
> But yeah, we are trying to take the Dredger a little bit out of the spotlight as an all-use vessel and more as a utility vessel, as all the new kws variants will definitely take the spotlight for multipurpose.

Ah, no, that's the opposite of what I want to accomplish with the KWS variants.  Nor do I want them to be the spotlight.
Overall I agree with Zitchas in that this should be something of a Korath baccy, though I'd rather it be a *different* ship than a straight-up upgrade, and don't like the idea of there *being* a "best ship," and in that way I totally agree with Quyykk.

Re Zitchas:
> This should be the ship that lags behind while the warships blaze ahead to engage, preferably engaging only from a distance. (I'd rather like it if we could get the Korath some 400mm railguns like the Aurorans made good use of in Escape Velocity Nova. That would be the ideal sort of weaponry for the Dredger's preferred combat role.

So... Heavers?

> As I see it right now, the Dredger as-is occupies something of a sweet spot in that it is big enough to be a solo ship, but not so big that it can't be risked flying without a fleet...  But that's no reason to erase the Dredger's versatility in the meantime.

I agree with the above...

> Contrast with the KWS and other huge ships that, frankly, I don't see the Korath ever risking going anywhere remotely dangerous without a large fleet of escorts. Too valuable. Ultra-useful, yes, and they'll put the Dredger permanently in the shadows when/if the player ever manages to get one.

...but *vehemently* disagree with this.  The KWS are not that big, really - there are other, if few, similarly sized ships - aside from the upcoming F variant which won't be leaving Exile space in normal use.  I think you're thinking too much in terms of their *current* role, rather than where we intend to take them with the introduction of the Type F, Refuge, Refabricator, and Haven.

A-E, meanwhile, will be specialised, stripped-down ships that take on specific roles, and while they may be excellent at those specific roles, none of those roles are going to involve moving quickly, and the Dredger should be able to out-perform each of them in at least one role they are not optimised for, and do it while moving faster.

> We also don't need to field super-optimized ships. It is entirely feasible to leave the ship _capable_ of fielding that offensive hardware, without ever having a variant that actually _has_ all that offensive hardware. We're writing the Korath lore here, all we need to do is say that the Korath value these ships too highly to allow them to be outfitted for strict offensive combat, and always appoint/select/elect/whatever captains to fly them that prefer either balanced builds or defensive measures.

This makes no sense.  They value their ships, but not enough to provide them with optimal protection?  The vibe I've been aiming for is quite the opposite, that Korath ship should, as much as is possible with the Korath's own tech, be highly optimised.  They can't afford to lose a ship because no-one thought its people were worth using up that last 17 tonnes for more shielding or something.

Once obtained by the player, they may optimise still further upon them by incorporating tech from other factions.

Reply:
Ah okay, forgive me for the confusion. Ultimately, I'm with warp_core on the nerfs, mainly the "battleship" capability, and more of a shift into a utility-only vessel (mining, invasion *support*, etc.). 

I agree with most of the changes/edits Zitchas wants to buff it back up, but I'm still very iffy on changing the weapon space back up. If we do, I would be okay with it on the condition that we keep the variant edits the same, and use the space given back for other things besides weaponry. 

As a side-note......I had a thought. If Zitchas likes the idea of Heavers, I could almost see, instead of KWS's, Ravagers, or Refabricators, the Dredger could be the premiere ship for Heavers. I see both turrets on the back two pods using them as default. They aren't super offensive turrets and are great artillery from afar. 

We can save Blaze-Pikes, Infernos, and Firestorms as more featured on the aforementioned aggressive ships (but the Refuge and Haven should still have Heavers). Perhaps a single KWS variant or two that use Heavers, but if going this route, I actually like the idea that the Dredger especially uses them as a support ship.

If we do agree on this Heaver route, I still would like us to commit Zitchas's changes (with the feedback) before I go and mess with variants again.

Reply:
I did some testing ahead of time, actually. I implemented the bunk and mass reverts. For outfit/weapon, I implemented warp_core's compromise suggestion of 675/240, but increased the weapon space to 242 for this next part. Finally, I tested as if we were to choose the Heaver Support ship I mentioned as a possibility, with more Fuel Processors to match. Here's what the variants are looking like now!

Default Variant:
![image](https://user-images.githubusercontent.com/93169396/176981074-e137ef56-47e2-42a2-a1e5-f1fcb076cc1f.png)
Fits nicely. 

Civilian Variant:
![image](https://user-images.githubusercontent.com/93169396/176981092-f42f95aa-8f54-4123-9c9f-fecb17f8aac7.png)
Also fits nicely.

Digger (Mining) Variant:
![image](https://user-images.githubusercontent.com/93169396/176981126-9f72e521-9e92-4ba4-a6e6-3999b8c5aae6.png)
Looking great. Leftover weapon space from preferring all diggers, of course.

Ground Assault Variant:
![image](https://user-images.githubusercontent.com/93169396/176981149-d9dde821-bbea-41f7-bb04-9f7572f9d7ca.png)
Looks good! Added a third banisher for good measure.

The Crippled Variant obviously works as it's a broken down default variant.

Shunt-Strike, I'm not so sure quite yet. Do we still want to keep it 4 Shunt-Focused, or add the Heavers and go down to 2 Shunt as well? 

Thoughts?

Reply:
> This makes no sense. They value their ships, but not enough to provide them with optimal protection? The vibe I've been aiming for is quite the opposite, that Korath ship should, as much as is possible with the Korath's own tech, be highly optimised. They can't afford to lose a ship because no-one thought its people were worth using up that last 17 tonnes for more shielding or something.

Oh, no, they should definitely have reasonable or perhaps even optimal protection. I'm just saying that doesn't necessarily involve (and in fact is probably *opposite*) being outfitted for optimum offensive capability. 

> If we do, I would be okay with it on the condition that we keep the variant edits the same, and use the space given back for other things besides weaponry.

I'm 100% in favor of this. Weapon space (just like engine space) is just a measure of the maximum stress structural components can use. there is zero reason that any ship should **need** to "max these out."

> Ah okay, forgive me for the confusion, but I very much agree that this should be an "alternate Bactrian" rather than "better Bactrian". Ultimately, I'm with warp_core on the nerfs, mainly the "battleship" capability, and more of a shift into a utility-only vessel (mining, invasion support, etc.).

I'm good with this being an "alternate" Bactrian rather than a "better" one. That being said, the current Bactrian is a cruiser, not a battleship. It's massively more agile than the Dredger can ever hope to be, and has significantly more firepower capability in the form of 4 additional gunports. So the fact that the Dredger feels like a "Battleship" style *is*, to me, actually a significant difference. 

But I guess the point is this: It sounds like we're all agreed that the Dredger should be an alternative Bactrian as far as role goes. Right now, though, it is significantly worse in:
- mobility, 
- weapon slots, 
- weapon capacity, 
- engine capacity, 
- outfitting capacity, and 
- unavoidable cost to operate. 

It's better in 
- hp, 
- cargo capacity, and 
- bunks.

I'll admit, I flat out *like* the fact that it doesn't have gunports. That's totally fitting and appropriate for it to not have guns, and I don't think that should change. That also guarantees that the Dredger will always have 40% less weapons than the Bactrian could have. (by weapon count). Likewise for the mobility. This isn't an agile ship in any way.

Reply:
Well, I don't agree that it *should* be an alternate Bactrian, but I do acknowledge that that's what people seem to *want*.

I'll point out that the Dredger beats the Bactrian in:

 - fighter bays
 - turret mounts
 - fully expanded outfit space
 - spawn rate / availability via capture / without having to complete a morally questionable storyline

I'd be happy to give it more engine space.